_posts/2021-04-13-working-with-legacy-code.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title: "Working with legacy code"
+date: 2021-04-13 12:00:00 +0000
+categories: legacy
+---
+## What is legacy code?
+There is some debate about what exactly constitutes “legacy code”, but it is generally older code which is difficult to maintain. We expand below on what makes code difficult to maintain and suggest a general approach to improving it.
+
+## What makes code difficult to maintain? 
+There are many articles written about software best practices and how to make code easier to maintain.  Example features of legacy code include:
```

Comment:
I suggest deleting (or moving) "There are many articles written about software best practices and how to make code easier to maintain."  - It doesn't add much, and with the previous paragraph starting with "there is some debate about...", it reads a little like we're trying to carefully qualify everything.

Reply:
(We were talking about adding some links to other articles, which might be a better way of putting this.)

Reply:
I've deleted the above line about "there are many articles..."